MIGRATION.md

```diff
@@ -116,9 +117,26 @@ The Parse ObjC SDK will be phased out in the future in favor of the more modern
 
 # Known Issues
 
-These issues below are especially important to consider before migrating.
+The following issues are important to consider before migrating.
 
-- After migrating an app to the Parse Swift SDK, launching the app will create a new `Installation` object with a new `installationId`. It will appear as if the app had been uninstalled and then reinstalled, even though it was only updated with the new Parse Swift SDK.
+## Installation
+
+After migrating an app to the Parse Swift SDK, launching the app will create a new `_Installation` object with a new `installationId`. It will appear as if the app had been uninstalled and then reinstalled, even though it was only updated with the new Parse Swift SDK.
```

Comment:
I believe there should be a clear distinction between `converting` and `migrating`. Converting an app from using the Objective-C SDK to Swift SDK shouldn’t assume migration. A migration typically entails some formal process or method(s) that are intentionally called to complete migration. If a developer wants to migrate a `PFUser` from the Objective-C Keychain to the Swift SDK Keychain, the developer needs to intentionally call [ParseUser.loginUsingObjCKeychain](https://parseplatform.org/Parse-Swift/release/documentation/parseswift/parseuser/loginusingobjckeychain(options:)/). Similarly, if the developer wants to migrate a `PFInstallation` from the Objective-C SDK, they need to intentionally call [ParseInstallation.become](https://parseplatform.org/Parse-Swift/release/documentation/parseswift/parseinstallation/become(_:copyentireinstallation:options:)). This Swift SDK was never intended to be a drop-in replacement for the Objective-C SDK, yet the way this migration document is structured tends to imply this is the case, it is not.

I don’t recall ever seeing comments or documentation about other Client SDK’s (Objective-C, JS, and Flutter) that work for IOS and other Apple OS’s state they can automatically access a ParseUser or ParseInstallation from another client SDK or imply they can be used interchangeably. If the other SDK’s aren’t held to this type of standard and don’t have comments or docs in their repos stating this, then why should the Swift SDK? In addition, these types of things should not be labeled as `issues` or `bugs` in this SDK.

> launching the app will create a new `_Installation` object with a new `installationId`.

In most of the client SDK’s, I don’t see `_Installation` used. `_Installation` is typically used on the server and in the dashboard. This comment is misleading as it makes it seem like the Swift SDK creates a new installation on the server. It does not do this. The installation is only local on the devices keychain. 

> It will appear as if the app had been uninstalled and then reinstalled, even though it was only updated with the new Parse Swift SDK.

This comment implies `drop-in replacement` which I addressed earlier. The Swift SDK isn’t a drop-in replacement and verbiage that eludes to such should be removed and/or modified as it’s misleading.

Reply:
It covers the same iOS ecosystem, so people will migrate. That involves a process, which is described here.

Reply:
It doesn't change much whether we call this a migration guide or a conversion guide - the issues stay the same. Let's stay with "migration" as that is maybe a more common term and it's already used to reference this document.

Reply:
It doesn't change much whether we call this a migration guide or a conversion guide - the issues stay the same. Let's stay with "migration" as that is maybe a more common term and it's already used to reference this document.

The standpoint you represent here goes beyond the terminology. I invite you to have a separate chat about this, because I see some fundamental discrepancies between your standpoint and developer/business expectations.

It's important that we get this migration info out to the developers, so I will consider this resolved. If you have any specific rewording suggestions please suggest.